docs/processing/preprocessing.md

```diff
@@ -49,38 +52,70 @@ We avert that session-wise *fMRIPrep*'s processes run into race conditions by pr
+        to the reportlet at hand. 
         As such, we encourage you to jump back and forth between visualizations as much as needed.
 
-
-### Anatomical preprocessing assessment
-
-- [ ] Assess the "Anatomical conformation" section and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#anatomical-conformation).
-- [ ] Assess the mosaic showing the calculated brain mask and brain tissue segmentation, and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#brain-mask-and-brain-tissue-segmentation-of-the-t1w).
-- [ ] Visualize the spatial normalization flickering mosaic, and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#spatial-normalization-of-the-anatomical-t1w-reference). Flickering between the subject and the template space is active while hovering your mouse on the mosaic area.
-- [ ] Assess the surface reconstruction mosaic, and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#surface-reconstruction).
 ### Assessment of fMRI Preprocessing
 
-- [ ] Scrutinize the textual summary and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#textual-summary).
-- [ ] Visualize the T2* map mosaic, and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#t2-map).
-- [ ] Check the T2* gray-matter intensity histogram, and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#t2-gray-matter-values).
-- [ ] Visualize the co-registration flickering mosaic, apply the [exclusion criteria](qaqc-criteria-preprocessed.md#alignment-of-functional-and-anatomical-mri-data).
+- [ ] Open each *fMRIPrep* functional report on a current Web Browser (*Google Chrome* is preferred).
+
+#### QCT
+!!! note "We are checking QCT fMRI first and in principle not excluding QCT scans"
+    Except if the image is [extremely distorted](qaqc-criteria.md#task-fmri-exclusion-criteria), we are not excluding QCT scans because we will leverage those  
+    images to evaluate the quality of fMRI scans and derived constructs throughout the whole analysis pipeline. We are however going through the *MRIQC* reports 
+    to train our eye, anticipate issues that might be affecting RSfMRI and flag the corresponding sessions.
+
+- [ ] Go through the report section of the QCT
+    - [ ] Scrutinize the textual summary and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#textual-summary).
+    - [ ] Visualize the T2* map mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#t2-map).
+    - [ ] Check the T2* gray-matter intensity histogram, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#t2-gray-matter-values).
+    - [ ] Visualize the co-registration flickering mosaic, apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#alignment-of-functional-and-anatomical-mri-data).
+        Flickering between T1w and BOLD images is active while hovering your mouse on the mosaic area.
+    - [ ] Skip the next reportlets and go to the BHT section
```

Comment:
If this is the same below, please merge and indicate that you sequentially do this on the QCT and then BHT.